### DISCUSSION TITLE
how to disable highlight color words

### DISCUSSION THREAD

**Discussion by lone1y-51** [NvChad/NvChad#1871] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11805860/228825198-79f57f7d-931e-4964-b651-614d1253b940.png)
i want to disable this backgroud color. but i cant find options.

**Comment by siduck** [comment, marked as answer]
hi @lone1y-51 
![image](https://user-images.githubusercontent.com/59060246/228830561-e5caabac-31c4-4571-a27e-0a5d3cf7280b.png)

 so just override the opts of nvim-colorizier
 
 ```lua
 {
 "NvChad/nvim-colorizer.lua",
 opts = {
          user_default_options = { 
              names = false 
            }
    }
 }

**Reply by lone1y-51** [reply]
thanks
